src/Modelling/PetriNet/Reach/Deadlock.hs

```diff
@@ -319,8 +318,6 @@ try conf = do
       ts
       (Modelling.PetriNet.Reach.Deadlock.capacity conf)
   return $ do
-    -- Filter out nets with isolated nodes
-    guard $ not $ hasIsolatedNodes n
```

Comment:
@marcellussiegburg, why is it safe to remove this guard here?

I know that `Modelling.PetriNet.Reach.Roll.net` has acquired a `not . hasIsolatedNodes` check, and also  `Modelling.PetriNet.Reach.Roll.conn` has acquired a `not (null vor) || not (null nach)` check.

But neither of these two functions appears to be used from here, not even indirectly. So, why can the above guard simply be dropped?